.github/workflows/deploy_to_aws.yml

```diff
@@ -228,6 +229,14 @@ jobs:
           push: true
           tags: ${{ vars.ECR_REGISTRY }}/${{ vars.IMAGE_NAME }}-database-anonymizer-${{ steps.get_env_name.outputs.lowercase }}:${{ env.GIT_COMMIT }}
 
+      - name: Build and push Anonymize Database Dump Docker image to ECR for ${{ vars.ENV_NAME }}
```

Comment:
@roninzp What happens here, I mean every deployment it will push anonymize db dump to ECR?

Reply:
No. At this stage, a docker image is built with a service that takes a dump from a shared anonymous snapshot and sends it to S3